framework/src/modules/chain/logic/init_transaction.js

```diff
@@ -163,25 +203,7 @@ class Transaction {
 			transactionJSON.asset = Object.assign(transactionJSON.asset, asset);
 		}
 
-		return this.jsonRead(_.omitBy(transactionJSON, _.isNull));
-	}
-
-	storageRead(raw) {
-		if (!raw.id) {
-			return null;
-		}
-
-		const rawData = _.omitBy(raw, _.isNull);
-
-		return this.jsonRead(rawData);
-	}
-
-	jsonRead(rawTx) {
-		const TransactionClass = this.transactionClassMap.get(rawTx.type);
-		if (!TransactionClass) {
-			throw new Error('Transaction type not found.');
-		}
-		return new TransactionClass(rawTx);
+		return this.fromJson(_.omitBy(transactionJSON, _.isNull));
 	}
 }
 
```

Comment:
This file export a class `Transaction` but the file name `init_transaction` suggest it do some actions.

Reply:
There is an issue for that already.

REF.: https://github.com/LiskHQ/lisk-sdk/pull/3440#discussion_r278441522